PocketKit/Sources/SharedPocketKit/UserDefaultsKey.swift

```diff
@@ -0,0 +1,102 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import Foundation
+import SwiftUI
+
+public extension UserDefaults {
+    enum Key: String, CaseIterable {
+        case hasAppBeenLaunchedPreviously = "hasAppBeenLaunchedPreviously"
+        case recentSearches = "Search.recentSearches"
+        case toggleAppBadge = "AccountViewModel.ToggleAppBadge"
+        case appBadgeToggle = "Settings.ToggleAppBadge"
+        case legacyUserMigration = "com.mozilla.pocket.next.migration.legacyUser"
+        case dateLastRefresh = "HomeRefreshCoordinator.dateLastRefreshKey"
+        case lastRefreshedTagsAt = "lastRefreshedTagsAt"
+        case lastRefreshedArchiveAt = "lastRefreshedArchiveAt"
+        case lastRefreshedSavesAt = "lastRefreshedSavesAt"
+        case listSelectedSortForSaved = "listSelectedSortForSaved"
+        case listSelectedSortForArchive = "listSelectedSortForArchive"
+        case readerFontSizeAdjustment = "readerFontSizeAdjustment"
+        case readerFontFamily = "readerFontFamily"
```

Comment:
@dskuza in a followup can we add comments for the intended use of each key? While I know due to pocket experience what readerFontFamily is, others may not know right away.

Reply:
:+1: that'll be a cross-team effort, but a good idea